### ISSUE TITLE
Whois bugged again? Prefix is not automatically handled by Send()

### ISSUE THREAD

**Issue by vegax87** [DanielOaks/oragono#85]
I tested your latest rev (5e6ac670105eed7e3379a58338e5ceb74f9149b1) and after you removed the : prefix, Whois command is bugged again and channels names don't appear anymore.

**Comment by DanielOaks** [comment]
Hmm, that's sketchy. Could I please ask you to run this command, to make sure the newest ver of girc-go and all are being used:
```
go get -u github.com/DanielOaks/girc-go
```

When running `WHOIS` locally, I can see this result:
```
10:16:49 <-- testnet WHOIS dan dan
10:16:49 --> testnet :lol.u 311 dan dan ~d n * :Dan's Real Name
10:16:49 --> testnet :lol.u 319 dan dan :@#itdc @#ex1 @#ex8 @#otherchan @#services @#a @#pypsd.control @#twitter-test @#testchan
10:16:49 --> testnet :lol.u 313 dan dan :is a cool dude
10:16:49 --> testnet :lol.u 317 dan dan 0 1484352948 :seconds idle, signon time
10:16:49 --> testnet :lol.u 318 dan dan :End of /WHOIS list
```

That 319 reply line seems to match `RPL_WHOISCHANNELS` ([defs](http://defs.ircdocs.horse/defs/numerics.html#rpl-whoischannels-319)/[1459](https://tools.ietf.org/html/rfc1459#section-6.2)). If you're seeing something different in terms of raw traffic then that's dodgy.

I'd be interested to see which client isn't managing this properly, mind letting me know which client it is so I can investigate?

**Comment by DanielOaks** [comment]
...If you only have a single channel then the prefix won't be applied, as it's not required. If your client is silly and doesn't respect sane rules wrt getting the final parameter then that may also be causing it to die here. In which case I can go submit an issue the dev's way.

**Comment by vegax87** [comment]
my client is AdiIRC v2.6.32

**Comment by DanielOaks** [comment]
Thanks, I'll take a look and see what's up here

**Comment by vegax87** [comment]
Yep, if is only 1 channel the prefix is not sent and the channel list is empty

**Comment by DanielOaks** [comment]
Heh, love it. Alright I should be able to implement a hacky workaround here, and in the meantime I'll reach out to the AdiIRC dev so they can parse this properly. Thanks for letting me know!

**Comment by DanielOaks** [comment]
Cool, issue created on their side over here: https://dev.adiirc.com/issues/3202

I'll investigate just manually adding a space at the end in all cases. I think I remember Mammon or something doing similarly to work around this bug in clients, but I'll take a look regardless. Thanks for bringing it up :)

**Comment by DanielOaks** [comment]
And I've pushed a fix in 2de273462b3a8ba57a35b55b6f4cfb5853b63967 (same thing Mammon does). Have confirmed that it works fine for AdiIRC so this should be all good!

Thanks again for opening this issue, it's much appreciated!
